### ISSUE TITLE
Empty files should pass with insert_final_newline = true

### ISSUE THREAD

**Issue by vrevelas** [ec4j/editorconfig-maven-plugin#43]
Empty files, such as `__init__.py` cause errors like `__init__.py@1,1: Insert lf - violates insert_final_newline = true, reported by org.ec4j.linters.TextLinter` when using a .editorconfig file like
```
root = true

[*]
end_of_line = lf
insert_final_newline = true
```

Adding a newline to these empty files as a workaround suppresses the error from this plugin, but then fails pylint checks (trailing-newlines check).

Excluding __init__.py files is also not ideal, as they are often empty, but sometimes contain code that should be checked.

I think the correct behaviour here would be to ignore empty files when checking for trailing newlines.

**Comment by ppalaga** [comment]
This is an interesting corner case. Thanks for the report, @vrevelas. I asked about that in the editorconfig project https://github.com/editorconfig/editorconfig/issues/416

**Comment by ppalaga** [comment]
Fixed by upgrading to editorconfig-linters 0.1.2 https://github.com/ec4j/editorconfig-maven-plugin/pull/47/commits/131eb5100dd32d720410bd08e202d0a2bff6c6d3

**Comment by ppalaga** [comment]
Fixed in the just released version 0.0.11. Thanks again for the report.
